layers/+emacs/org/packages.el

```diff
@@ -175,10 +175,18 @@ Will work on both org-mode and any mode that accepts plain html."
       (spacemacs/set-leader-keys-for-major-mode 'org-mode
         "'" 'org-edit-special
         "c" 'org-capture
+
+        ;; Clock
+        ;; These keybindings should match those under the "aok" prefix (below)
         "Cc" 'org-clock-cancel
-        "Ci" 'org-clock-in
+        "Cd" 'org-clock-display
+        "Cg" 'org-clock-goto
+        "Ci" 'org-clock-in-last
```

Comment:
I see you replaced `org-clock-in` with `org-clock-in-last`, is this intentional?

Reply:
I'm glad you pointed this out. I'm of two minds here: it more closely matches the `aok`-prefix keybindings (which I think is positive), but removes a common action from the keybindings. What do you think about the following change, under both prefixes:
- `CI`: `org-clock-in-last`
- `ci`: `org-clock-in`

Reply:
Yes it makes sense to have them match and I like the solution.

Reply:
In fact @sdwolfz what would you think about moving `aok` to `aoC` for more consistency as well?

Reply:
I've made both changes: `aok -> aoC` and `ci`/`cI`. I've also updated the `CHANGELOG` to be accurate.